eventually/static/src/containers/profile/ProfileEdit.js

```diff
@@ -6,6 +6,7 @@ import FileUpload from '../fileUpload/FileUpload';
 import TextField from 'material-ui/TextField';
 import RaisedButton from 'material-ui/RaisedButton';
 import DatePicker from 'material-ui/DatePicker';
+import {blueGrey800,orange500} from 'material-ui/styles/colors';
```

Comment:
space `{blueGrey800, orange500}`

Reply:
I dont need this colors so I deleted them.